data/names/ru.json

```diff
@@ -3089,26 +3101,32 @@
       "Арсе",
       "Артеага",
       "Артур",
+      "Аруда",
+      "Арфа",
       "Арчер",
       "Арчулета",
       "Асеведо",
       "Аскью",
-      "Аткинсон",
       "Аткинс",
+      "Аткинсон",
       "Ахмад",
       "Ахмед",
       "Ашер",
       "Аяла",
       "Бабб",
       "Баггетт",
       "Байерс",
+      "Байнум",
+      "Бак",
       "Бака",
       "Бакли",
       "Бакнер",
+      "Бакон",
```

Comment:
This would have to come from the name Buckon in order to receive this spelling. Otherwise, it's more likely to be a transcription of the last name Bacon, such as with Francis Bacon: [Фрэнсис Бэкон](https://ru.wikipedia.org/wiki/%D0%91%D1%8D%D0%BA%D0%BE%D0%BD,_%D0%A4%D1%80%D1%8D%D0%BD%D1%81%D0%B8%D1%81).

Reply:
[Kevin Bacon ](https://en.wikipedia.org/wiki/Kevin_Bacon)is [Кевин Бэйкон](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D0%B9%D0%BA%D0%BE%D0%BD,_%D0%9A%D0%B5%D0%B2%D0%B8%D0%BD), though.